internal/admin/dashboard/templates/index.html

```
@@ -384,27 +384,45 @@ <h3 x-text="guardrailFormMode === 'edit' ? 'Edit Guardrail' : 'Create Guardrail'
                 <label class="alias-form-field">
                     <span>User Path</span>
                     <input type="text" class="filter-input" placeholder="/team/alpha" x-model="guardrailForm.user_path" aria-label="Guardrail user path">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)